src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int LOTTERY_BALL = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lottery = new Lottery[LOTTERY_BALL];
+        for (int i = 0; i < lottery.length; i++) {
+            lottery[i] = new Lottery();
+            Ball ball = lottery[i].getRandomBall();
+            System.out.println(ball.toString());
+        }
```

Comment:
You should have only one example of Lottery
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < LOTTERY_BALL; i++) {
            System.out.println(lottery.getRandomBall().toString());
        }
```